docs/course-information.md

```diff
@@ -7,43 +7,51 @@ description: >-
+This course is aimed at master-level students from Computer Science, Simulation Sciences and related fields, but is also open to PhD students or anyone else interested. We expect you to have some knowledge in software development, basic tools and programming. However, we also give you extra information or point to additional resources if you have the feeling you need to catch up with some topic.
 
-*An example*
-We will shortly touch on [Git](https://git-scm.com/)
-- You are a [Git](https://git-scm.com/) expert? That is great.
-- You have used Git before, but basically use the same 5 commands the same time? That is fine and a good foundation to
-- You had heard about Git some time in a lecture, but you have never used it? You will be good to go with the examples in the lectures and the references presented there.
+*An example*: In the lecture, we shortly touch on [Git](https://git-scm.com/)
 
-In any case would like you to tell us about your experiences with Git and how you use it.
+- You are a Git expert? This is great. We try not to bore you.
+- You have used Git before, but you basically use the same 5 commands all the time? This is fine and a good foundation to start.
+- You had heard about Git some time in a lecture, but you have never used it? You are good to go with the examples in the lectures and the references presented there, but you probably have to catch up a bit.
 
 ## Resources
 
-All teaching resources will be made available online on GitHub and this homepage.
+All teaching resources are (will be) available online on GitHub and on this homepage.
 
-We encourage **you** to extend these resources, fix mistakes or update the materials.
+We encourage **you** to help us extend these resources and fix mistakes.
 
 ## Assignments and lab work.
 
-There will be two types of assignments. We will have smaller lab work that will help you to get familiar with the presented software engineering tools
+There are two types of assignments. We have smaller weekly lab work, which helps you to get familiar with the presented software engineering concepts and tools.
+
+
+Parallel to the lab work, you work on an individual challenge, where you apply the learned concepts and tools with the ultimate goal to contribute to a large-scale community simulation software package. The challenge is structured in three parts, whereas each part is completed by a short presentation of the intermediate results in class:
+
+1. You get acquainted with the basic functionality of a large-scale simulation software package (such as FEniCS, PETSc, TRILINOS, or SU2) by studying tutorials and documentation (first quarter of the course)
```

Comment:
Should the list of software be consistent with the chapter "Content of the course" where it is "preCICE, DuMUX, TRILINOS, or FEniCS"?